docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Creates a Docker image that is ready to execute tes compliance suite
+
+FROM ubuntu:latest AS build-env
```

Comment:
Starting from a Python base image could reduce the build time.

Reply:
I looked around for a ubuntu image with 3.8 python and couldnt really find one unless it was published by an individual. If you know of an image I could use, that'd be great! thankfully this doesnt take too long so if we cant find one, that should be okay.

Reply:
Indeed, the official Python images (https://hub.docker.com/_/python) are built on either Debian or Alpine, not Ubuntu.

I typically use one of the `slim` Debian builds, pinned to an exact or a minor version. If you'd always like the latest Python 3.8, you could use `3.8-slim-bullseye`, built on the current stable Debian release Bullseye, or `3.8-slim-buster`, built on the previous stable Debian release Buster. See here for more info on the Debian releases: https://wiki.debian.org/DebianReleases. Alternatively, you could also use `3.8-slim`, which currently corresponds to `3.8-slim-bullseye` but would automatically use the next stable Debian release once it becomes available.

As Ubuntu is built on Debian and I don't see anything Ubuntu-specific in the Dockerfile, the Debian images would probably work just fine for this use case. But if for some policy or other reason you really need Ubuntu (or I missed some Ubuntu dependency), then of course going with Ubuntu and manually installing Python is totally fine.

I would certainly not recommend using an unofficial Ubuntu/Python image.

Reply:
funny how I was so focused on ubuntu->python and didnt think the other way around! appreciate your patience and suggestions!